extension/src/lib/extensionHolder.js

```diff
@@ -0,0 +1,58 @@
+/*globals fluid */
+
+"use strict";
+
+var gpii = fluid.registerNamespace("gpii");
+var chrome = chrome || require("sinon-chrome");
+
+fluid.defaults("gpii.chrome.extensionHolder", {
+    gradeNames: "fluid.modelComponent",
+    extensionId: "",
+    members: {
+        extensionInstance: null
+    },
+    model: {
+        extensionEnabled: undefined
+    },
+    invokers: {
+        setEnabled: {
+            funcName: "gpii.chrome.extensionHolder.setEnabled",
+            args: ["{that}", "{arguments}.1"]
+        },
+        getEnabled: {
+            funcName: "gpii.chrome.extensionHolder.getEnabled",
+            args: "{that}.extensionInstance"
+        }
```

Comment:
Does the extensionInstance have any events for model changes that we could hook into. It seems strange to have get/set methods on the component that don't actually effect the component's model. Also perhaps these should be renamed to "setExtensionInstanceModel" or something like that. Also is it necessary to have the "get" invoker, as the value is easily accessible off of the `extensionInstance` member object.